### ISSUE TITLE
Statements indicating responsibility that do not name an agent 

### ISSUE THREAD

**Issue by deborahjleslie** [rbms-bsc/DCRMR#98]
The RBMS PS editorial group made the decision to make a deviation both from RDA and its predecessor standards, to transcribe statements indicating some action upon the text (e.g., translated into English ...) but without naming a person or body as Other title information instead of as a Statement of responsibility. I couldn't find the reason for this decision documented, but memory vaguely reconstructs it as a deviation for rare materials reasons, a change I was very much in favor of and probably introduced. The [examples ](http://access.rdatoolkit.org/rdachp2_rda2-4753.html)given in the current RDA are enough different from the sort of verbose and intermingled OTI and SoR information commonly found in early printed materials that I can imagine how such a deviation could be supported.   

Am now second-guessing that decision. Considerations:

- New RDA continues to be [unambiguous](https://beta.rdatoolkit.org/en-US_ala-a39c4ac1-03cb-351b-8868-f64d7dd16de4/p_nrn_wlp_zdb) about it, but there are no examples to give the intention context. 

- This instruction appears in the broader element [Statement of responsibility](https://beta.rdatoolkit.org/en-US_ala-a39c4ac1-03cb-351b-8868-f64d7dd16de4), not [Statement of responsibility relating to title proper](https://beta.rdatoolkit.org/en-US_ala-93929207-c91d-3970-b325-af8fd9dd2956)

- My understanding of broader elements is that treatment of them applies also to the narrower elements and are not repeated there. 'Agent not named' is not addressed in SoR relating to title proper, so presumably the broader instruction applies 

- However, 'Agent not named' is also not addressed in [Statement of responsibility relating to edition](https://beta.rdatoolkit.org/en-US_ala-dcdb7f9e-3b2f-3819-8c00-1a5e6a1339ed); yet our [instructions ](https://docs.google.com/document/d/1juWF_PhS9YZP-M-MGTflUPjfrS2YZt7BKNzyZx---DQ/edit) follow conventional practice, and we are instructed not to record statements with unnamed agents as a statement of responsibility for the edition. 

- I still think a valid argument could be made for the essential difference between clear statements of responsibility (e.g., by a group of students with a Korean resource person; with a spoken commentary by the artist) and the sort of actions performed upon the text using verb phrases that we find in early materials. But since it's a substantial departure from our own conventions and does deviate from RDA, it's worth having fresh eyes in this new group of people address it anew.

**Comment by honormoody** [comment]
TL;DR I don't think there is a problem here, and I don't believe the draft text that Deborah links to deviates from new RDA. If no agent is identified in some fashion, the statement must be part of designation of edition, not statement of responsibility related to edition. 

Deborah's understanding that the narrower elements inherit the treatment of the broader element is correct, "A metadata statement that records a value of an element subtype of an RDA element has latent conformance because a value of an element subtype is also a value of an element supertype. The element subtype may not be an RDA element." (From [Well-formed RDA](https://beta.rdatoolkit.org/en-US_topic_v3l_3wv_xmb/p_dcz_q5f_plb).) I believe the narrower elements all include an option that makes this explicit; for Statement of responsibility relating to edition it is [this option](https://beta.rdatoolkit.org/en-US_ala-dcdb7f9e-3b2f-3819-8c00-1a5e6a1339ed/div_wtj_djh_qhb). 

The Beta Toolkit is unambiguous that you may choose to implement the [option ](https://beta.rdatoolkit.org/en-US_ala-a39c4ac1-03cb-351b-8868-f64d7dd16de4/div_hxf_hqp_zdb)to include an SOR that does not name the agent; it does not require it. However, the SOR must still reference an agent to meet the definition of an SOR, "A statement that identifies, and indicates the function of, an agent who is responsible for a work or its expression that is embodied by a manifestation. A statement of responsibility may include words or phrases that are neither names nor linking words."

**Comment by elizhobart** [comment]
Thanks for weighing in, Honor! Just to be 100% clear, a statement that doesn't name an agent such as "translated into English" may, under the Beta Toolkit, be recorded as statement of responsibility by implementing the option. Recording it as other title information, however, would not be correct. Is that accurate?

**Comment by honormoody** [comment]
No, “translated into English” without specifying an agent cannot be an SOR, as an SOR must identify an agent, even if it does not include a name.

**Comment by elizhobart** [comment]
Thanks, Honor! I think the confusion is that DCRM(B) 1E12 considered statements such "translated into English" to be statements of responsibility because they implied that someone did the translating. With RDA, a phrase such as "translated into English by a scholar" would constitute statement of responsibility, but "translated into English" does not.

**Comment by honormoody** [comment]
I see--I was looking only at the updated google doc version that Deborah linked to above.

**Comment by deborahjleslie** [comment]
Thanks, Honor and Elizabeth, for following up. This discussion confirms my gut reaction that verbs indicating an action upon the text but without referring to an agent are not statements of responsibility.  Or to put it positively, statements of responsibility must _name or refer_ to an agent.

I don't have access to DCRM(B) predecessors at the moment, but can confidently affirm that the 'translated out of Latin into English' example in DCRM(B) was carried over from from earlier rules.

**Comment by elizhobart** [comment]
Thanks, Deborah. It sounds like we're all on the same page here, so I'll go ahead and close this issue.
